databricks/koalas/namespace.py

```diff
@@ -1093,23 +1093,19 @@ def pd_read_excel(io_or_bin, sn):
 
         def read_excel_on_spark(pdf, sn):
 
+            if isinstance(pdf, pd.Series):
```

Comment:
nit: I think it'd be better to use `pdf_or_pser` or something if `pdf` can also be a `Series`. but not urgent now :)